src/components/ProjectCard/types.ts

```diff
@@ -0,0 +1,9 @@
+export type Props = {
+    status: 'active' | 'cancelled' | 'inactive';
```

Comment:
Can you please try with enum
```suggestion
enum ProjectStatus {
     active = 'active',
     cancelled = 'cancelled',
     inactive = 'inactive',
 } 
export type Props = {
    status: ProjectStatus;
```

Reply:
@kimnovak Good idea. I agree.

Reply:
Thanks! 
I've changed this, too. 👍